### ISSUE TITLE
Fix issue from Pagination screen reader accessibility review for navigation landmark

### ISSUE THREAD

**Issue by rsmithadhoc** [department-of-veterans-affairs/vets-design-system-documentation#2114]
## Description

Correct issue with `va-pagination` found during screen reader accessibility review. 

## Details

- **Must:** Navigation landmark does not have a name. When multiple `<nav>` elements are on the same page, they should be given a unique name for accessibility.
    - **Recommendation:** Add an `aria-label="pagination"` to the `<nav>` element.

## Tasks
- [ ] Fix the accessibility issue

## Acceptance Criteria
- [ ] Accessibility issue addressed
